interop/tapir/src/main/scala/caliban/interop/tapir/TapirAdapter.scala

```diff
@@ -124,12 +143,12 @@ object TapirAdapter {
     requestCodec: JsonCodec[GraphQLRequest],
     mapCodec: JsonCodec[Map[String, Seq[String]]],
     responseCodec: JsonCodec[GraphQLResponse[E]]
-  ): PublicEndpoint[(Seq[Part[Array[Byte]]], ServerRequest), StatusCode, GraphQLResponse[E], Any] =
+  ): PublicEndpoint[(Seq[Part[Array[Byte]]], ServerRequest), TapirResponse, GraphQLResponse[E], Any] =
     endpoint.post
       .in(multipartBody)
       .in(extractFromRequest(identity))
       .out(customJsonBody[GraphQLResponse[E]])
-      .errorOut(statusCode)
+      .errorOut(statusCode.and(stringBody).and(headers).mapTo[TapirResponse])
```

Comment:
How making a function for `statusCode.and(stringBody).and(headers).mapTo[TapirResponse]`? It's repeated a few times.

Reply:
Done